tests/test_run_topostats.py

```diff
@@ -74,6 +76,82 @@ def test_process_scan_both(regtest, tmp_path, process_scan_config: dict, load_sc
 # def run_topostats_arguments() -> arg.ArgumentParser:
 
 
+@pytest.mark.parametrize(
+    "image_set, expected",
+    [
+        ("core", False),
+        ("all", True),
+    ],
+)
+def test_save_cropped_grains(
+    tmp_path: Path, process_scan_config: dict, load_scan_data: LoadScans, image_set, expected
+) -> None:
+    """Tests if cropped grains are saved only when image set is 'all' rather than 'core'."""
+    process_scan_config["plotting"]["image_set"] = image_set
+    process_scan_config["plotting"] = update_plotting_config(process_scan_config["plotting"])
+
+    print(f"image set: {process_scan_config['plotting']['image_set']}")
+
+    img_dic = load_scan_data.img_dic
+    _, _ = process_scan(
```

Comment:
Brilliant, having this test `process_scan()` is going to make it nice and easy for me to move tests around when I rebase on `main` because as I mentioned I've moved `process_scan()` out to its own sub-module. :+1: